packages/keychain/tsconfig.json

```diff
@@ -1,37 +1,23 @@
 {
+  "extends": "../../tsconfig.react.json",
   "compilerOptions": {
     "baseUrl": "./src",
     "rootDir": ".",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "declaration": true,
-    "target": "ES2022",
-    "module": "esnext",
-    "strict": false,
-    "jsx": "preserve",
-    "allowJs": true,
-    "skipLibCheck": true,
-    "forceConsistentCasingInFileNames": true,
-    "incremental": true,
-    "esModuleInterop": true,
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "downlevelIteration": true,
-    "noEmit": true,
     "types": [
       "@emotion/react/types/css-prop"
-    ]
+    ],
+    // TODO(#262): Enable these options (too many errors for this scope)
```

Comment:
A bit too much errors to fix in this scope but should be fixed asap. A lot of type errors has been surpressed.